ConfigurationSystem/Client/Utilities.py

```diff
@@ -486,11 +486,9 @@ def getDBParameters( fullname, defaultHost = 'localhost',
   'db' and 'queueSize'
   """
 
-  fullname = fullname
```

Comment:
fullname is only used in the line below, so I don't understand your comment about this specific one.

Regarding the other default values, then there's maybe a mistake: was getOption() or getValue() to be used? If it's getValue(), then I undestand, but as the code is done the default passed ones are actually never used.